build

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I'd like to leave in for now just for the sake of consistency. It's unlikely to run on any system with an unusual /bin/bash location.